Ginger/Ginger/SolutionWindows/AddApplicationPage.xaml.cs

```diff
@@ -103,18 +111,101 @@ private void OKButton_Click(object sender, RoutedEventArgs e)
             {
                 mSolution.ApplicationPlatforms = new ObservableList<ApplicationPlatform>();
             }
+            var name = variableName.Text.Trim();
+            var description = variableDescription.Text.Trim();
+            if (string.IsNullOrEmpty(name))
+            {
+                NameError.Visibility = Visibility.Visible;
+                return;
+            }
+
+            eUserMsgSelection msgSelection = eUserMsgSelection.None;
+
+            if (!this.isSolutionNew)
+            {
+                msgSelection = Reporter.ToUser(eUserMsgKey.ShareApplicationToEnvironment);
+            }
+
+
+
             foreach (ApplicationPlatform selectedApp in SelectApplicationGrid.Grid.SelectedItems)
             {
                 mSolution.SetUniqueApplicationName(selectedApp);
+
```

Comment:
Add validation for duplicate name, ask user to provide unique name in case of duplicate name

Reply:
Added the required validation